site/docs/services/open-lineage-services.md

```diff
@@ -20,7 +20,7 @@ Following sequence explains how different services interact in and out the open
 
 2. [Asset Lineage OMAS](/services/omas/asset-lineage/overview) feeds the lineage warehouse by selecting and sending the instance types suitable for lineage. It retrieves asset related metadata and completes the asset context portion of the graph. Once ready, this lineage context graph structured as vertices and edges is sent on Lineage Output Topic for further processing and preservation.
 
-3. **Lineage event listener** consumes asset context events and uses technology connector such as [Lineage Janus Connector](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector) to persist these lineage elements building up the large lineage graph. 
+3. **Lineage event listener** consumes asset context events and uses technology connector such as [Lineage Janus Connector](/connectors/runtime/open-lineage-janus-connector) to persist these lineage elements building up the large lineage graph. 
```

Comment:
isn't the correct link /connectors/governance-daemon/open-lineage-janus-connector (?)

Reply:
fixed